website/blog/2018-05-29-jest-23-blazing-fast-delightful-testing.md

```diff
@@ -0,0 +1,124 @@
+
+![Snapshot Property Matchers](/jest/img/blog/23-snapshot-matchers.png)
+
+See the updated `toMatchSnapshot` [docs](https://facebook.github.io/jest/docs/en/expect.html#tomatchsnapshotpropertymatchers-snapshotname) or the Property Matcher [guide](https://facebook.github.io/jest/docs/en/snapshot-testing.html#property-matchers) for more information.
+
+## Custom Asynchronous matchers
+
+We now support asynchronous matchers with `expect.extends`! Async matchers return a Promise so that you can `await` for the matcher to resolve. As an example:
+
+![Custom asynchronous matchers in action](/jest/img/blog/23-async-matchers.png)
+
+Docs available [here](https://facebook.github.io/jest/docs/en/expect.html#expectextendmatchers).
+
+## Custom Asymmetric Matchers
+
+Asymmetric matchers are a great tool to use when you only care about asymmetric equality. For example, when the left side is expected to be an object with some subset of properties on the right, instead of an exact match. Jest provides a number of asymmetric matchers out of the box, and in this release we're introducing Custom Asymmetric Matchers.
+
+![Custom asymmetric matchers in action](/jest/img/blog/23-asymmetric-matchers.png)
+
+## Jest Each
+
+[@mattphillipsio](https://twitter.com/mattphillipsio) has donated the `jest-each` package to Jest Core (thanks Matt!). jest-each is a library inspired by [mocha-each](https://yarnpkg.com/en/package/mocha-each) and [Spock Data Tables](http://spockframework.org/spock/docs/1.1/data_driven_testing.html#data-tables) which allows you to define a table of test cases, and then run a test for each row with the specified column values. We support both array types and template literals for all flavors of `describe` and `test`. Docs are available [here](https://facebook.github.io/jest/docs/en/api.html#testeachtable-name-fn), and for those not on Jest 23 yet, we're still publishing [jest-each](https://yarnpkg.com/en/package/jest-each) separately!
+
+![jest-each in action](/jest/img/blog/23-jest-each.png)
+
+Huge shout out to Prettier for [fixing](https://github.com/prettier/prettier/pull/4423) the template string formatting so quickly ([see 1.13 Prettier post](https://prettier.io/blog/2018/05/23/1.13.0.html#format-new-describeeach-table-in-jest-23-4423-by-ikatyang))!
+
+## New Matchers
+
+We only add matchers to core if we believe they will be useful to a large amount of people in the Jest community, and leave the majority of matchers to the community (see [jest-extended](https://yarnpkg.com/en/package/jest-extended)). Some matchers make the cut into core, and Jest 23 adds:
+
+* nthCalledWith
```

Comment:
something about the fact that mocks now track returned and thrown value would be nice